telegram_bot/handlers/common.py

```
@@ -1,22 +1,60 @@
 
-@router.message(CommandStart())
-async def handle_start(message: Message) -> None:
+@router.message(StateFilter(None), Command("me"))
+async def handle_me(message: Message) -> None:
+    if message.from_user is None:
+        return
+    try:
+        linked_user = ensure_authorized_telegram_user(message.from_user.id)
+        profile = fetch_profile(linked_user.jwt_token)
+    except BotAuthError as exc:
+        await message.answer(f"Не авторизован: {exc}")
```

Ruff (0.15.5)

[warning] String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)